### PR TITLE
added StoreConverter

### PR DESCRIPTION
**Pull request by grigveronika**

### CODE REVIEW

File: src/main/java/internship/mbicycle/storify/converter/StoreConverter.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    public Store fromStoreDTOToStore(StoreDTO storeDTO) {
+        if (storeDTO == null) {
+            return null;
+        }
+        Store store = new Store();
+        store.setId(storeDTO.getId());
+        store.setStoreName(storeDTO.getStoreName());
+        store.setDescription(storeDTO.getDescription());
+        store.setAddress(storeDTO.getAddress());
+        return store;
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/converter/StoreConverter.java:24]: use builder

File: src/main/java/internship/mbicycle/storify/converter/StoreConverter.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+@Controller
+@RequiredArgsConstructor
+public class StoreConverter {
+
+ //   private final ProfileConverter profileConverter; //
+
+    public Store fromStoreDTOToStore(StoreDTO storeDTO) {
+        if (storeDTO == null) {
+            return null;
+        }
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/converter/StoreConverter.java:18]: dto cant be null here

**Review by Estrin-MBicycle — CHANGES_REQUESTED** [review]

File: src/main/java/internship/mbicycle/storify/converter/StoreConverter.java (hunk 1 above)
Comment (grigveronika) [reply src/main/java/internship/mbicycle/storify/converter/StoreConverter.java:24]: done

File: src/main/java/internship/mbicycle/storify/converter/StoreConverter.java (hunk 2 above)
Comment (grigveronika) [reply src/main/java/internship/mbicycle/storify/converter/StoreConverter.java:18]: done

**Pull request merged**